eng/Generate.ps1

```diff
@@ -52,15 +52,10 @@ $configurationPath = Join-Path $testServerDirectory 'readme.tests.md'
 $testServerSwaggerPath = Join-Path $repoRoot 'node_modules' '@microsoft.azure' 'autorest.testserver' 'swagger'
 $testNames =
     'additionalProperties',
-    #'azure-composite-swagger',
     #'azure-parameter-grouping',
-    #'azure-report',
-    #'azure-resource',
-    #'azure-resource-x',
     #'azure-special-properties',
     'body-array',
     'body-boolean',
-    #'body-boolean.quirks',
```

Comment:
What's in all these quirks swaggers?

Reply:
Lol.
![image](https://user-images.githubusercontent.com/17788297/76038062-85a2d780-5efd-11ea-9679-ae061dadc88f.png)

Reply:
Should we delete them from the test server? Along with other swaggers without tests?

Reply:
There might be v2 generators that use them to test things. We have no idea. 🐢